harpy/_validations.py

```
@@ -386,12 +354,8 @@
     with open(regioninput, "r", encoding="utf-8") as fin:
         badrows = []
         idx = 0
-        while True:
-            line = fin.readline()
-            if not line:
-                break
-            else:
-                idx += 1
+        for line in fin:
+            idx += 1
```

Ruff

Use `enumerate()` for index variable `idx` in `for` loop

(SIM113)